components/FlexibleHeader/src/private/MDCFlexibleHeaderView+Private.h

```diff
@@ -36,4 +36,9 @@
  */
 - (void)extractTopSafeAreaInset;
 
+/**
+ The length of the top safe area guide.
```

Comment:
height? Maybe reconsider the name of the property as well?

Reply:
Done.